components/org.wso2.carbon.identity.conditional.auth.functions.analytics/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/analytics/internal/AnalyticsFunctionsServiceComponent.java

```diff
@@ -103,6 +103,44 @@ protected void activate(ComponentContext context) {
         }
     }
 
+    @Activate
+    protected void activate(ComponentContext context) {
+
+        try {
+            JsFunctionRegistry jsFunctionRegistry = AnalyticsFunctionsServiceHolder.getInstance()
+                    .getJsFunctionRegistry();
+            CallAnalyticsFunction callSiddhi = new CallAnalyticsFunctionImpl();
```

Comment:
\`\`\`suggestion
LOG.debug("Registering CallAnalyticsFunction in JsFunctionRegistry");
\`\`\`